app/shop.py

```diff
@@ -0,0 +1,33 @@
+import dataclasses
+import datetime
+
+from typing import Union, Any
+
+
+@dataclasses.dataclass
+class Shop:
+    name: str
+    location: list
```

Comment:
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contains string elements has typehint list[str]